pyseir/ensembles/ensemble_runner.py

```diff
@@ -221,6 +225,38 @@ def _load_model_for_fips(self, scenario='inferred'):
         model.run()
         return model
 
+    def _model_ensemble(self, override_params=None, N_samples=5000):
```

Comment:
Consider making this method public since you call `er.model_ensemble`:
```suggestion
    def model_ensemble(self, override_params=None, N_samples=5000):
```